pkg/machine/define/vmfile_windows.go

```diff
@@ -0,0 +1,33 @@
+package define
+
+import (
+	"errors"
+	"fmt"
+	"golang.org/x/sys/windows"
+	"os"
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Delete removes the machinefile symlink (if it exists) and
+// the actual path
+func (m *VMFile) Delete() error {
+	if m.Symlink != nil {
+		if err := os.Remove(*m.Symlink); err != nil && !errors.Is(err, os.ErrNotExist) {
+			logrus.Errorf("unable to remove symlink %q", *m.Symlink)
+		}
+	}
+	if err := os.Remove(m.Path); err != nil && !errors.Is(err, os.ErrNotExist) {
+		if errors.Is(err, windows.ERROR_SHARING_VIOLATION) {
+			fmt.Println("BAUDE BAUDE BAUDE BAUDE")
+			cmd := exec.Command("handle", []string{m.GetPath()}...)
+			if err := cmd.Run(); err != nil {
+				fmt.Println(err)
+			}
```

Comment:
this will not show any command output by default
you either need to connected stdout/stderr (not sure if this works on windows) or use something like cmd.CombinedOutput() and then print the result